app/models/recommendation.rb

```diff
@@ -24,4 +24,12 @@ class Recommendation < VersionedRecord
 
   validates :title, presence: true
   validates :reference, presence: true, uniqueness: true
+
+  SUPPORT_LEVELS = {
```

Comment:
can we just have an integer? not sure we need to specify the different levels like this

Reply:
in other words: i am happy for the database to support any level.